### PR TITLE
Add test script helping to spawn topology test env.

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: scripts/multinode.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+	sudo ovs-vsctl add-port br-int vm1-eth0
+
+        if [ ! -z "$2" ]; then
+		sudo ovs-vsctl add-br br-tun
+		sudo ovs-vsctl add-port br-int patch-int -- set interface patch-int type=patch
+		sudo ovs-vsctl add-port br-tun patch-tun -- set interface patch-tun type=patch
+		sudo ovs-vsctl set interface patch-int option:peer=patch-tun
+		sudo ovs-vsctl set interface patch-tun option:peer=patch-int
+		sudo ovs-vsctl add-port br-tun gre0 -- set interface gre0 type=gre options:remote_ip=$1
```
Comment (nplanel) [inline scripts/multinode.sh]: You should change gre0 interface name by something like mn-gre0

File: scripts/simple.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+#   ns2 ip/mask: create a second ns
+
+
+function start() {
+	set -x
+
+	sudo ovs-vsctl add-br br-int
+
+	sudo ip netns add vm1
+	sudo ip l add vm1-eth0 type veth peer name eth0 netns vm1
```
Comment (nplanel) [inline scripts/simple.sh]: s/ip l/ip link/g
s/ip a/ip add/g

File: scripts/multinode.sh (hunk 1 above)
Comment (safchain) [inline scripts/multinode.sh]: well ok, even if I don't know what mn does mean

File: scripts/simple.sh (hunk 2 above)
Comment (safchain) [inline scripts/simple.sh]: Done

Comment (safchain) [inline scripts/simple.sh]: BTW ip address will be better

File: scripts/multinode.sh (hunk 1 above)
Comment (nplanel) [inline scripts/multinode.sh]: mn stand for multinode

**Pull request merged**